packaging/helm-charts/helm3/strimzi-kafka-operator/values.yaml

```diff
@@ -50,6 +50,15 @@ rbac:
 serviceAccountCreate: yes
 serviceAccount: strimzi-cluster-operator
 
+# If you are using the grafana dashboard sidecar,
+# you can import some default dashboards here
+dashboards:
+  enabled: false
+  label: grafana_dashboard
+  namespace: ~
+  labelValue: "1"
```

Comment:
@scholzj  the ConfigMap containing dashboard configuration should have the label of "grafana_dashboard" with the value of "1", in order to be scraped from grafana's sidecar.

Reply:
wouldn't be better to use something more reasonable than just magic number? Or at least add an explanation comment here or to readme?

Reply:
Thanks @leewoobin789. That would explain. I wonder if it would make sense to have also some `labels` option for defining additional labels (similarly to `annotations`). But maybe that can be added later if needed.

Reply:
In theory I've addressed this with an updated patch just now.